android/app/src/main/java/com/google/codeu/chatme/view/adapter/UserListAdapter.java

```diff
@@ -97,5 +111,24 @@ private void setHolderPicture(String picUrl) {
                         .into(this.civUserPic);
             }
         }
+
+        @Override
+        public void onClick(View view) {
+
```

Comment:
David pointed out in one of the earlier PRs that it's better that we just pass the created object to our presenter, so if and when a field in the model class changes, we don't need to change the signature of presenter methods.